ec2/spark_ec2.py

```diff
@@ -643,12 +654,14 @@ def setup_cluster(conn, master_nodes, slave_nodes, opts, deploy_ssh_key):
 
     # NOTE: We should clone the repository before running deploy_files to
     # prevent ec2-variables.sh from being overwritten
+    repo_branch="{r} -b {b}".format(r=opts.spark_ec2_git_repo, b=opts.spark_ec2_branch)
+    print "Cloning spark-ec2 scripts from {rb} on master...".format(rb=repo_branch)
```

Comment:
@nchammas Just realised that "master" is actually correct here, since it refers to the fact that this is happening on the master node (as per original log message and consistent with other log messages), rather than being a reference a branch name. 

I'll add it back in (but keep the .../tree/... change) unless you have any objections.